packages/sdk-js/package.json

```diff
@@ -17,6 +17,7 @@
   "engines": {
     "node": ">=10"
   },
+  "sideEffects": false,
```

Comment:
I understand this is needed for tree shaking, however ... We do have side effects, right? (Adding properties to the global `window`object).